src/opendr/perception/object_detection_2d/ssd/ssd_learner.py

```diff
@@ -0,0 +1,688 @@
+# Copyright 2020-2021 OpenDR European Project

```

Comment:
I think that the corresponding documentation is missing for this learner (https://github.com/tasostefas/opendr_internal/tree/master/docs/reference). See an existing template (e.g., https://github.com/tasostefas/opendr_internal/blob/master/docs/reference/lightweight-open-pose.md)